server/config/config.go

```diff
@@ -289,6 +292,7 @@ func DefaultConfig() *Config {
 			IAVLCacheSize:       781250, // 50 MB
 			IAVLDisableFastNode: false,
 			IAVLLazyLoading:     false,
+			ABCIClientType:      "committing", // [AGORIC]
```

Comment:
No way to reference the "other" default value ?

Reply:
Not where it is now - circular imports. Let me see if I can do the reference the other way - or at least add cross-reference comments.

Reply:
Done. Now the options and default are defined in one place - the config package. (The server/types package seems like the better place, but it imports server/config.)

Reply:
D'oh! It would help if I actually used the constant I just defined. Actually done now.